### DISCUSSION TITLE
How to get 'Github Campus Expert' highlight?

### DISCUSSION THREAD

**Discussion by rodukov** [community/community#32331] [Other Feature Feedback, Questions, & Ideas] [answered]
> How to get 'Github Campus Expert' highlight?
![img](https://user-images.githubusercontent.com/101352977/189179199-e9cd8a77-209f-4af4-b350-e11755c22341.png)

**Comment by radar-blip** [comment]
How to become [Campus Expert](https://github.com/campus-experts)

**Reply by rodukov** [reply]
Are you sure it's possible to get it in principle? **I think it's the same situation with her as with 'discussion answered'**. Do you have any github profiles to prove its existence?

**Comment by shaido987** [comment, marked as answer]
You can apply here: https://githubcampus.expert/training

If you are selected then you need to go through their training program, after that, you should be able to get the highlight on your profile page.  
Some examples of campus experts:
- https://github.com/juanpflores  
- https://github.com/ROODAY  
- https://github.com/PandelisZ
- https://github.com/yashovardhan
